packages/fullscreenView/components/FullscreenView.tsx

```diff
@@ -24,56 +25,65 @@ export interface FullscreenViewProps {
   onClose: (event?: React.SyntheticEvent<HTMLElement>) => void;
   /** The base of the `data-cy` value. This is used for writing selectors in Cypress. */
   cypressSelectorBase?: string;
+  /** The css max width around the header contents  */
+  containerMaxWidth?: string;
```

Comment:
❓ Should we allow a certain set of values here or just any string?

Reply:
as a user I think I would want to be able to set this to any value, otherwise it loses usefulness pretty fast. There could be all kinds of situations in which I wouldn't want to use a particular breakpoint